portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,43 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+/* Servlet that returns json of commentaries statistics*/
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
+  /* Some hardcoded examples*/
+  String [] commentList = {"Hi! I'm Mark", "Hello, my name is Nancy :3", "Good morning/afternoon/evening! I'm Alex"};
+  String [] nameList = {"Mark", "Nancy", "Alex"};
+  String [] ratingList = {"4/5", "5/5", "0/5"};
+  int listLength = commentList.length;
+
+  /* Processing of GET request to /comment page, json dtring will be returned*/
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    String json = "{";
+    json += convertToJsonList("comment", commentList);
+    json += ",";
+    json += convertToJsonList("name", nameList);
+    json += ",";
+    json += convertToJsonList("rating", ratingList);
+    json += "}";
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  //Converter to json format "name":[item1, item2, item3, ...]
+  private String convertToJsonList(String type, String [] listOfObj) {
```

Comment:
You should never implement JSON (or other encoding) yourself unless you have no other option. Manual encoding is very risky and error-prone. Please use https://github.com/google/gson to do this. It was already added as a dependency to the pom.xml file above. See https://www.tutorialspoint.com/gson/gson_object_serialization.htm or https://www.tutorialspoint.com/convert-java-object-to-json-using-the-gson-library-in-java.